hugegraph-dist/src/assembly/travis/install-hbase.sh

```diff
@@ -2,17 +2,18 @@
 set -ev
 
 TRAVIS_DIR=`dirname $0`
+HBASE_VERSION=2.0.2
```

Comment:
add the reason to commit message, and correct typo "upgraph"

Reply:
done